commit 1d170d3f9afce53b748acdbc0612758bba39aa35
Author: Eric Blake <eblake@redhat.com>
Date:   Wed Aug 1 11:00:35 2012 -0600

    build: commit to 0.10.0 release naming
    
    With 0.10.0-rc0 out the door, we are committed to the next version
    number.
    
    * src/libvirt_public.syms (LIBVIRT_0.9.14): Rename...
    (LIBVIRT_0.10.0): ...to this.
    * docs/formatdomain.html.in: Fix fallout.
    * src/openvz/openvz_driver.c (openvzDriver): Likewise.
    * src/remote/remote_driver.c (remote_driver): Likewise.

diff --git a/docs/formatdomain.html.in b/docs/formatdomain.html.in
index c834577..f97c630 100644
--- a/docs/formatdomain.html.in
+++ b/docs/formatdomain.html.in
@@ -780,7 +780,7 @@
         CPU model will fail. Supported values for <code>fallback</code>
         attribute are: <code>allow</code> (this is the default), and
         <code>forbid</code>. The optional <code>vendor_id</code> attribute
-        (<span class="since">Since 0.9.14</span>)  can be used to set the
+        (<span class="since">Since 0.10.0</span>)  can be used to set the
         vendor id seen by the guest. It must be exactly 12 characters long.
         If not set the vendor id of the host is used. Typical possible
         values are "AuthenticAMD" and "GenuineIntel".</dd>
diff --git a/src/libvirt_public.syms b/src/libvirt_public.syms
index 5004182..e3ba119 100644
--- a/src/libvirt_public.syms
+++ b/src/libvirt_public.syms
@@ -544,7 +544,7 @@ LIBVIRT_0.9.13 {
         virDomainSnapshotRef;
 } LIBVIRT_0.9.11;
 
-LIBVIRT_0.9.14 {
+LIBVIRT_0.10.0 {
     global:
         virDomainGetHostname;
         virConnectRegisterCloseCallback;
diff --git a/src/openvz/openvz_driver.c b/src/openvz/openvz_driver.c
index 4cc7f46..48b5219 100644
--- a/src/openvz/openvz_driver.c
+++ b/src/openvz/openvz_driver.c
@@ -2170,7 +2170,7 @@ static virDriver openvzDriver = {
     .domainIsUpdated = openvzDomainIsUpdated, /* 0.8.6 */
     .isAlive = openvzIsAlive, /* 0.9.8 */
     .domainUpdateDeviceFlags = openvzDomainUpdateDeviceFlags, /* 0.9.13 */
-    .domainGetHostname = openvzDomainGetHostname, /* 0.9.14 */
+    .domainGetHostname = openvzDomainGetHostname, /* 0.10.0 */
 };
 
 int openvzRegister(void) {
diff --git a/src/remote/remote_driver.c b/src/remote/remote_driver.c
index 9354cb4..b9e2127 100644
--- a/src/remote/remote_driver.c
+++ b/src/remote/remote_driver.c
@@ -5366,7 +5366,7 @@ static virDriver remote_driver = {
     .domainGetDiskErrors = remoteDomainGetDiskErrors, /* 0.9.10 */
     .domainSetMetadata = remoteDomainSetMetadata, /* 0.9.10 */
     .domainGetMetadata = remoteDomainGetMetadata, /* 0.9.10 */
-    .domainGetHostname = remoteDomainGetHostname, /* 0.9.14 */
+    .domainGetHostname = remoteDomainGetHostname, /* 0.10.0 */
 };
 
 static virNetworkDriver network_driver = {
